src/com/dabomstew/pkrandom/constants/Gen7Constants.java

```diff
@@ -210,13 +216,67 @@ public static int getPokemonCount(int romType) {
     private static List<Integer> setupAllHeldItems() {
         // We intentionally do not include Z Crystals in this list. Adding Z-Crystals to random trainers should
         // probably require its own setting if desired.
-        List<Integer> list = new ArrayList<>();
-        list.addAll(Gen6Constants.allHeldItems);
+        List<Integer> list = new ArrayList<>(Gen6Constants.allHeldItems);
         list.addAll(Arrays.asList(adrenalineOrb, electricSeed, psychicSeed, mistySeed, grassySeed));
         list.addAll(Arrays.asList(terrainExtender, protectivePads));
         return list;
     }
 
+    public static final List<Integer> generalPurposeConsumableItems = initializeGeneralPurposeConsumableItems();
+
+    private static List<Integer> initializeGeneralPurposeConsumableItems() {
```

Comment:
I was a little surprised not to see the pinch berries here. Does the confusion chance still make it not worth it?

Additionally, while I haven't looked into this very much, it looks like Natures are defined on the trainer Pokemon data structure in Gen 7:
![image](https://user-images.githubusercontent.com/12245827/116799807-73e0a400-aab0-11eb-8525-252278d58cdc.png)

We actually read and write the nature in Gen7RomHandler (doesn't look like we *ever* change it though), so *if* I'm understanding this correctly, we could give Gen 7 trainer Pokemon pinch berries that *wouldn't* confuse them. This probably involves a decent amount of effort, though, so I totally understand if you don't want to do it or want to make it a follow-up PR.

Reply:
I had forgotten about the healing berries. I added them back in. I'm going to hold off on matching the berries to the natures for now.